.github/workflows/tests.yml

```diff
@@ -70,12 +70,10 @@ jobs:
         run: inv -r faasmcli/faasmcli libs.fake
       - name: "Run codegen"
         run: inv -r faasmcli/faasmcli codegen.local
-      - name: "Run WAMR codegen"
-        run: inv -r faasmcli/faasmcli codegen demo echo --wamr
+      - name: "Run python codegen"
+        run: inv -r faasmcli/faasmcli python.codegen
```

Comment:
This step in the `codegen.local` task was failing so I've added it explicitly here